WebApp/.gitignore

```diff
@@ -38,3 +38,5 @@ dist
 
 # Mac files
 .DS_Store
+.package.json
```

Comment:
package.json file cannot be in gitignore, because it keeps track of all JS modules used it project. You would not be able to install dependencies without it.

package-lock.json cannot be ignored neither....